src/term/mod.rs

```diff
@@ -1336,7 +1336,7 @@ impl Term {
 
     #[inline]
     pub fn should_exit(&self) -> bool {
-        tty::process_should_exit() || self.should_exit
+        self.should_exit
```

Comment:
The `use crate::tty;` import should be removed.

Reply:
It's actually needed for `tty::isconpty` on windows - I'll pop a `#[cfg(windows)]` on there.

Reply:
Thanks, it's mainly about the compile-time warning. Including it conditionally makes sense here.